frontend/app/(dashboard)/documents/page.tsx

```
@@ -447,26 +406,177 @@ export default function DocumentsPage() {
+
+  return (
+    <Dialog open onOpenChange={(isOpen) => !isOpen && onClose()}>
+      <DialogContent className="max-h-[85vh] overflow-y-auto">
+        <DialogHeader>
+          <DialogTitle>Share Document</DialogTitle>
+          <DialogDescription>Select who are you sharing "{document.name}" with.</DialogDescription>
+        </DialogHeader>
+        {document.textContent ? (
+          <div
+            className="prose border-muted min-h-0 grow overflow-y-auto rounded-md border p-4 text-black"
+            dangerouslySetInnerHTML={{ __html: document.textContent ?? "" }}
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)